source/common/ssl/context_config_impl.cc

```diff
@@ -135,12 +153,7 @@ ServerContextConfigImpl::ServerContextConfigImpl(const envoy::api::v2::Downstrea
   // TODO(PiotrSikora): Support multiple TLS certificates.
   // TODO(mattklein123): All of the ASSERTs in this file need to be converted to exceptions with
```

Comment:
@PiotrSikora as long as you are in here, can you take a look at the rest of the asserts in this file? I think some of them could potentially happen/crash with native v2 config. I might be wrong, but could you take a look and potentially fix to throw/add tests? (Follow up is fine if that is better).

Reply:
Sure, I'll do in a separate PR.